_maps/thebackstreets/thebackstreets.yaml

```diff
@@ -0,0 +1,186 @@
+---
+name:
+  en: The Backstreets
+desc:
+  en:  The Backstreets on the edges of the map lead to eachother. The ones in the middle lead to the suits. Which roads will you take?
+ruleSet: Standard
+theme: DragonQuest
+initialCash: 2500
+targetAmount: 20000
+baseSalary: 300
+salaryIncrement: 250
+maxDiceRoll: 6
+frbFile1: TheBackstreets1
+frbFile2: TheBackstreets2
+frbFile3: TheBackstreets3
+frbFile4: TheBackstreets4
+background: traverseTown
+mapIcon: backstreets
+music:
+  download: 
+  - https://nikkums.io/cswt/BGM/thebackstreets.music.zip
+  - https://drive.google.com/file/d/1RwK5rUJhayHv4o-By0IpTIyW7AADd-0_/view?usp=sharing

```

Comment:
This one should be: https://drive.google.com/uc?export=download&confirm=1&id=1RwK5rUJhayHv4o-By0IpTIyW7AADd-0_

Reply:
Thanks so much for fixing those links! I'll add those corrections and resubmit the boards again soon.